### PR TITLE
feat: UI data stubs

### PR DESCRIPTION
**Pull request by andrii-uhryn-absa**

### CODE REVIEW

File: packages/frontend/src/fixtures/stubs.ts [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+      credentials: [
+        'did:indy:sovrin:staging:C279iyCR8wtKiPC8o9iPmb/anoncreds/v0/SCHEMA/e-KYC/1.0.0',
+      ],
+      id: '8fa665b6-7fc5-4b0b-baee-6221b1844ec8',
+      createdAt: '2023-11-10T14:43:36.083Z',
+      updatedAt: '2023-11-20T16:49:58.307Z',
+    },
+    {
+      name: 'Absa 2',
+      dids: ['did:indy:sovrin:staging:C279iyCR8wtKiPC8o9iPmc'],
```
Comment (niall-shaw) [inline packages/frontend/src/fixtures/stubs.ts:19]: I think we should use a different entity, with different did (should be unique per entity), logo domain etc.
Even the id is the same!

File: packages/frontend/src/fixtures/stubs.ts [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    {
+      schemaId:
+        'did:indy:sovrin:staging:C279iyCR8wtKiPC8o9iPmb/anoncreds/v0/SCHEMA/e-KYC/1.0.0',
+      name: 'Digital Identity',
+      createdAt: '2023-11-10T14:41:49.143Z',
+      updatedAt: '2023-11-20T16:49:09.838Z',
+    },
+    {
+      schemaId:
+        'did:indy:sovrin:staging:C279iyCR8wtKiPC8o9iPmc/anoncreds/v0/SCHEMA/e-KYC/1.0.0',
```
Comment (niall-shaw) [inline packages/frontend/src/fixtures/stubs.ts:42]: Again, perhaps a different schema?

File: packages/frontend/src/fixtures/stubs.ts [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+    domain: 'http://absa.co.za',
+    role: ['issuer'],
+    credentials: [
+      'did:indy:sovrin:staging:C279iyCR8wtKiPC8o9iPmb/anoncreds/v0/SCHEMA/e-KYC/1.0.0',
+    ],
+    invitationId: 'k4o74jb1u6bto0owh9si0sux',
+    id: '6a89a0fc-3e22-44d8-9d27-35c79b9f76c5',
+    createdAt: '2023-11-16T15:39:09.694Z',
+    updatedAt: '2023-11-16T15:39:09.694Z',
+    state: 'declined',
```
Comment (niall-shaw) [inline packages/frontend/src/fixtures/stubs.ts:81]:
```suggestion
    state: 'rejected',
```
We don't have state `declined`

File: packages/frontend/src/fixtures/stubs.ts [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+    emailAddress: 'alice.doe@example.com',
+    id: 'ib6o9ki9fssprzljkuth6fvj',
+    createdAt: '2023-11-16T15:26:00.362Z',
+  },
+  {
+    emailAddress: 'tom.doe@example.com',
+    id: 'f7vahm7zfx5sw24rb8i589yj',
+    createdAt: '2023-11-16T15:28:26.858Z',
+  },
+]
```
Comment (niall-shaw) [inline packages/frontend/src/fixtures/stubs.ts:127]: I guess two of these should be associated with the existing entities, i.e. `entityId: ___`

File: packages/frontend/src/fixtures/stubs.ts [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+      schemaId:
+        'did:indy:sovrin:staging:C279iyCR8wtKiPC8o9iPmb/anoncreds/v0/SCHEMA/e-KYC/1.0.0',
+      name: 'Digital Identity',
+      createdAt: '2023-11-10T14:41:49.143Z',
+      updatedAt: '2023-11-20T16:49:09.838Z',
+    },
+    {
+      schemaId:
+        'did:indy:sovrin:staging:Pmi5CXJbv4HM8YttyomMoW/anoncreds/v0/SCHEMA/e-KYC/1.0.0',
+      name: 'Digital Identity 2',
```
Comment (jakubkoci) [inline packages/frontend/src/fixtures/stubs.ts:43]: We should use real data as much as possible. What about Phone Number or Bank Account credential?

File: packages/frontend/src/fixtures/stubs.ts [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+      createdAt: '2023-11-10T14:41:49.143Z',
+      updatedAt: '2023-11-20T16:49:09.838Z',
+    },
+  ],
+}
+
+export const submissionsStub = [
+  {
+    name: 'John Doe',
```
Comment (jakubkoci) [inline packages/frontend/src/fixtures/stubs.ts:52]: This should be a company name, not a person.

File: packages/frontend/src/fixtures/stubs.ts [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+      role: ['issuer'],
+      credentials: [
+        'did:indy:sovrin:staging:C279iyCR8wtKiPC8o9iPmb/anoncreds/v0/SCHEMA/e-KYC/1.0.0',
+      ],
+      id: '8fa665b6-7fc5-4b0b-baee-6221b1844ec8',
+      createdAt: '2023-11-10T14:43:36.083Z',
+      updatedAt: '2023-11-20T16:49:58.307Z',
+    },
+    {
+      name: 'Example entity',
```
Comment (jakubkoci) [inline packages/frontend/src/fixtures/stubs.ts:18]: No examples, real companies pls :)

**Review by jakubkoci — COMMENTED** [review]
It's good Andrii, just use a real data pls. Absa, Standard Bank, ... It helps to understand the system and relations.

File: packages/frontend/src/fixtures/stubs.ts [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+        'did:indy:sovrin:staging:C279iyCR8wtKiPC8o9iPmb/anoncreds/v0/SCHEMA/e-KYC/1.0.0',
+        'did:indy:sovrin:staging:C279iyCR8wtKiPC8o9iPmb/anoncreds/v0/SCHEMA/e-KYC-mobile/1.0.0',
+        'did:indy:sovrin:staging:C279iyCR8wtKiPC8o9iPmb/anoncreds/v0/SCHEMA/e-KYC-residence/7.0.0',
+        'did:indy:sovrin:staging:C279iyCR8wtKiPC8o9iPmb/anoncreds/v0/SCHEMA/test-absa-bank-account/1.0.0',
+      ],
+      id: '8fa665b6-7fc5-4b0b-baee-6221b1844ec8',
+      createdAt: '2023-11-10T14:43:36.083Z',
+      updatedAt: '2023-11-20T16:49:58.307Z',
+    },
+  ],
```
Comment (jakubkoci) [inline packages/frontend/src/fixtures/stubs.ts:20]: Could we pls add at least Standard Bank as a verifier here? Or DIDx, they should have some did on sovrin ledger 🤔

File: packages/frontend/src/fixtures/stubs.ts [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+      createdAt: '2023-10-11T14:41:49.143Z',
+      updatedAt: '2023-10-21T16:49:09.838Z',
+    },
+  ],
+}
+
+export const submissionsStub = [
+  {
+    name: 'Absa',
+    dids: ['did:key:z6MkjBWPPa1njEKygyr3LR3pRKkqv714vyTkfnUdP6ToFSH5'],
```
Comment (jakubkoci) [inline packages/frontend/src/fixtures/stubs.ts:56]: Is the `did:key` supported did? Is it resolveable by trust registry? 🤔

**Review by jakubkoci — CHANGES_REQUESTED** [review]

File: packages/frontend/src/fixtures/stubs.ts [hunk 10]
```diff
@@ -0,0 +1,109 @@
...
+      ],
+      id: '8fa665b6-7fc5-4b0b-baee-6221b1844ec8',
+      createdAt: '2023-11-10T14:43:36.083Z',
+      updatedAt: '2023-11-20T16:49:58.307Z',
+    },
+    {
+      name: 'DIDx',
+      dids: ['did:indy:sovrin:staging:L9YosTGp7hDUcHLmTC8dE9'],
+      logo_url: 'https://didx.net/static/media/logo.d4f6a758.png',
+      domain: 'https://didx.net',
```
Comment (jakubkoci) [inline packages/frontend/src/fixtures/stubs.ts:24]: This is wrong company

logo https://www.safbc.co.za/logos/didx.png
domain https://www.didx.co.za/

**Review by jakubkoci — CHANGES_REQUESTED** [review]

File: packages/frontend/src/fixtures/stubs.ts [hunk 11]
```diff
@@ -0,0 +1,112 @@
...
+    invitationId: 's3gdwtyxt8w8dn87sbisxjn0',
+    id: '8f2917fa-139a-41eb-855e-245d6bed96df',
+    createdAt: '2023-11-16T15:24:31.630Z',
+    updatedAt: '2023-11-16T15:24:31.630Z',
+    state: 'approved',
+  },
+  {
+    name: 'DIDx',
+    dids: ['did:indy:sovrin:builder:L9YosTGp7hDUcHLmTC8dE9'],
+    logo_url: 'https://didx.net/static/media/logo.d4f6a758.png',
```
Comment (jakubkoci) [inline packages/frontend/src/fixtures/stubs.ts:87]: Still wrong

**Review by jakubkoci — CHANGES_REQUESTED** [review]

**Review by jakubkoci — APPROVED** [review]

**Pull request merged**